src/renderer/coremods/settings/pages/QuickCSS.tsx

```diff
@@ -22,6 +28,38 @@ type ThemeModule = {
   removeChangeListener: (listener: () => unknown) => unknown;
 };
 
+const PopoutModule = await webpack.waitForModule(
+  webpack.filters.bySource('type:"POPOUT_WINDOW_OPEN"'),
+);
+// eslint-disable-next-line @typescript-eslint/non-nullable-type-assertion-style
+const openPopout = webpack.getFunctionBySource(PopoutModule, "POPOUT_WINDOW_OPEN") as (
+  key: string,
+  render: ReactComponent<unknown>,
```

Comment:
I completely deleted the `ReactComponent` type quite some time ago for obvious reasons. You could pass any prop without type checking. Instead create a new type with the correct types.